translations/en.yml

```
@@ -222,8 +227,14 @@ src:
       Settings:
         Account:
           AccountSection:
+            '56029204': >-
+              Light theme is still in beta, some UI may still not be supported
+              (For example: public projects will not use light theme).
```

```suggestion
              Light theme is still in beta. Some UI may still not be supported
              (ie. public projects will not use light theme).
```
Sorry just small change..